### ISSUE TITLE
NOTIFY keyword not getting highlighted

### ISSUE THREAD

**Issue by kovacsvx** [hpcc-systems/vscode-ecl#159]
It appears NOTIFY is a key word, but it doesn't highlight as such.  I tried writing a module called notify and we had unexpected errors.

**Comment by GordonSmith** [comment]
@kovacsvx it is there and does get highlighted on my machine (the same as OUTPUT)...
https://github.com/hpcc-systems/vscode-ecl/blob/master/syntaxes/ecl.tmLanguage.json#L88

Do you have a sample of ECL where its not highlighting?

**Comment by kovacsvx** [comment]
IMPORT IADP.Util.Notify as Notify;

    sendSentinelWarningEmail() := MACRO
        Notify.AddAlert('hpcc_loader', . . .

We made the mistake of calling a library "notify", and eclcc
started complaining that Notify.AddAlert needed a '(', not a '.'.

After some head banging, we discovered it was a reserved word.
Bitmap below (if it gets through) to show that it didn't highlight.


[cid:image001.png@01D5D37E.56289D60]

Victor Kovacs
Systems Engineer, IADP
RELX Group
Reed Elsevier, 9443 Springboro Pike, Miamisburg OH  45342  M/S 21S52
Phone: 937-247-3434
vic.kovacs@relx.com<mailto:vic.kovacs@relx.com>
www.relx.com
